docs/source/guides/ursula_configuration_guide.rst

```diff
@@ -74,9 +82,34 @@ If your installation is non-functional, be sure you have the latest version inst
 
 
 
-2. Configure a new Ursula node
+3. Configure a new Ursula node
 --------------------------------
 
+With Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Recall the volume mounts: `-v <path to a directory on your computer>:/root/.local/share/` which is where your Nucypher node stores persistent data as well as accesses ipc with your locally running geth node.
+
+Execute the following commands (Ubuntu):
+
+.. code:: bash
+
+    export NUCYPHER_KEYRING_PASSWORD=<your keyring password>
+    export MY_IP=$(wget -q -O - ifconfig.me);
+    export NUCYPHER_WORKER_ADDRESS=<eth account checksum of your worker>
+    export NUCYPHER_STAKER_ADDRESS=<eth account checksum of your staker>
+    export NUCYPHER_WORKER_ETH_PASSWORD=<your eth account password>
+
+    # init your worker
+    docker run -v /home/ubuntu:/root/.local/share/ -e NUCYPHER_KEYRING_PASSWORD -it nucypher/nucypher:latest nucypher ursula init --provider /root/.local/share/geth/.ethereum/goerli/geth.ipc --poa --worker-address $NUCYPHER_WORKER_ADDRESS --staker-address $NUCYPHER_STAKER_ADDRESS --rest-host $MY_IP --sync
```

Comment:
**NOTE**: `--sync` is no longer a valid option for `nucypher ursula init` and `nucypher ursula run` (L107 below)

Reply:
https://github.com/nucypher/nucypher/pull/1432